core/app/modules/site-editor/assets/js/organisms/site-parts.js

```diff
@@ -27,12 +29,17 @@ const InfoButton = ( props ) => {
 						<p>{ props.tip }</p>
 					</div>
 				</section>
-				<section>
-					<h3>{ __( 'Watch Video', 'elementor' ) }</h3>
-					<div className="video-wrapper">
-						<iframe id="ytplayer" src={ props.video_url } frameBorder="0" />
-					</div>
-				</section>
+
+				{
```

Comment:
can you make it more readable ? (use if)

Reply:
updated